code/style.css

```diff
@@ -1,7 +1,325 @@
+  margin: 0; 
 }
+
+body { 
+  background-color: rgb(255, 255, 255);
+  font-family: "Montserrat";
+
+}
+
+a {
+  color: rgb(88, 88, 88);
+  text-decoration: none;
+}
+
+a:hover {
+  color: #00747c;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.header { 
+  display: flex;
+  justify-content: center; 
+  width: auto;
+
+  /* border: 3px solid green; */
+}
+
+video { /*  https://css-tricks.com/full-page-background-video-styles/ */
+  object-fit: cover;
+  width: 100%;
+  height: 400px;
+}
+
+h1 { 
+  text-align: center;
+  color: rgb(235, 233, 233);
+  text-shadow:2px 2px #4e4e4e;
+  font-size: 50px;
+  text-transform: uppercase;
+}
+
+h2 {
+  text-align: center;
+  color: rgb(235, 233, 233);
+  text-shadow:2px 2px #4e4e4e;
+  font-size: 30px;
+  text-transform: uppercase;
+  padding: 1rem;
+}
+
+.overlay {
+  position:absolute;
+  top:5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+  /* border:2px solid green; */
+}
+
+button {
+  padding:15px;
+  background-color: cadetblue;
+  border:none;
+  border-radius: 8px;
+  font-family: "Montserrat";
+  color:white;
+  cursor: pointer;
+}
+
+button:hover {
+  background: #00747c;
+  transition: 0.25s;
+}
+
+/* MAIN SECTION*/
+.container {
+  display: flex;
+  justify-content: center;
+  width: auto;
+  /* border:4px solid violet; */
+}
+
+/* FORM SECTION */
+
+/* fieldset and legend */ 
+fieldset {
+  margin: 2rem 0 ; 
+  padding: 1em 2em;
+  border: 1px solid rgb(110, 110, 110);
+  border-radius: 6px;
+}
+
+legend {
+  font-size:1.25em;
+  padding:0 .25em;
+  color: #999;
+}
+
+/*labels*/
+label {
+  display: block; /* places all element in a row*/ 
+  margin-top: 1em; /*adding space between elements in a form*/ 
+  color:rgb(75, 75, 75);
+}
+
+.checks label {
+  margin-top: 0; 
+}
+
+label:first-of-type {
+  margin-top:1px;
+}
+
+/*inputs */ 
+
+input {
+  padding: .5em;
+  border: 1px solid #999;
+}
+
+input[type="email"], input[type="text"] {
+  width: 15em;
+}
+
+/*radio buttons and checkboxes*/ 
+.options {
+  margin-bottom: 2em;
+  margin-left:1em;
+}
+
+.labels-radio {  
+  margin-bottom: 1em;
+}
+
+.checks {
+  margin-bottom:2em;
+  margin-top:3em;
+}
+
+.checks p {
+  margin:1em 0;
+}
+
+input[type="checkbox"] + label, input[type="radio"] + label {
+  display: inline-block;
+  padding-top:0;
+  margin-top: 0;
+}
+
+input[type="radio"] {
+  margin-left: 1.5em;
+  margin-top:0;
+}
+
+input[type="radio"]:first-of-type {
+  margin-left:0;
+}
+
+/*ACCORDION*/
+
+.acc-container {
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 2rem;
+  /* border:2px solid red; */
+}
+
+/* Style the buttons that are used to open and close the accordion panel */
+.accordion {
+  color: rgb(250, 250, 250);
+  cursor: pointer;
+  padding: 18px;
+  width: auto;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.4s;
+}
+
+.accordion::after {
+  content: "\002B";  
+  float: right;
+}
```

Comment:
Is this different to the one down on line 198? If so, how? A description would be appreciated to better understand the code.